src/css/application.css

```diff
@@ -342,6 +349,52 @@ body {
   }
 }
 
+/** @define instructions-editor */
+
+.instructions-editor {
+  display: flex;
+  flex-direction: column;
+  height: 100%;
+  background-color: var(--color-low-contrast-gray);
+}
+
+.instructions-editor__input {
+  padding: 1rem 1rem 0;
+  flex-grow: 1;
+  overflow-y: scroll;
+
+  /* postcss-bem-linter: ignore */
+  & textarea {
+    height: 100%;
```

Comment:
This doesn’t seem to be having the desired effect in Chrome 65—the editor is just a couple lines high. I find absolute positioning with all four sides pinned works wonders in this kind of situation ; )